### ISSUE TITLE
PCSX2 : about aliasing on moving

### ISSUE THREAD

**Issue by mondaychicken** [PCSX2/pcsx2#2853]
***PCSX2 version:***
pcsx2-v1.5.0-dev-2928-g1ae48db87-windows-x86

***PCSX2 options:***
Default Settings

***Plugins used:***
Default Settings

***Description of the issue:***
I don't know what I should call this aliasing. (texture shimmering or flickering )

But this aliasing seems to be called "specular aliasing".

Reference
https://www.youtube.com/watch?v=DI9-AlST4Uk

This event occurs almost on moving.

Bilinear and anisotropic filtering slightly help to remove this aliasing, but  this aliasing is made yet.

Screenshot (red circle is aliasing event places)

DOA2 hard core
![gsdx_20190212234733](https://user-images.githubusercontent.com/35862336/52690707-e46b7b00-2fa1-11e9-8795-16b0adcd767e.png)

Soul Calibur 2 
![gsdx_20190212235322](https://user-images.githubusercontent.com/35862336/52690724-f51bf100-2fa1-11e9-8d0b-bf96ca112052.png)

![gsdx_20190212235456](https://user-images.githubusercontent.com/35862336/52690744-09f88480-2fa2-11e9-999f-09c360452faf.png)

![gsdx_20190212235634](https://user-images.githubusercontent.com/35862336/52690752-1250bf80-2fa2-11e9-9fa8-25fff778fc9e.png)

Tales of Destiny
![gsdx_20190213145644](https://user-images.githubusercontent.com/35862336/52690766-1f6dae80-2fa2-11e9-99c6-e2a9d5bae6b3.png)

In screenshot it may not know well, but it will know this event on moving if you play the games.

Latest pcsx2 does not have 'msaa hack'.
But 'msaa hack' used to be helpful to remove the aliasing.

In short,  I wonder
this aliasing can be removed on pcsx2 ?

thanks.

***PC specifications:***
CPU: Intel haswell 4690
GPU: amd r9 280
OS: Windows 7 64 Bit

**Comment by lightningterror** [comment]
Latest accuracy changes on dx11 were not compatible with msaa so we removed it.

Anyway you can crank up the resolution more.

Closing this.

**Comment by MrCK1** [comment]
You could also use FXAA with PgUp

**Comment by Dreadmoth** [comment]
You could try Bilinear (Forced) texture filtering. It may result in some blur, but should reduce the texture aliasing. If some remains, try switching to the OpenGL (Hardware) renderer and enable Trilinear Filtering (under Advanced Settings and Hacks).
